compiler.go

```diff
@@ -537,24 +539,31 @@ func (c *Compiler) Compile(node parser.Node) error {
 				moduleName += ".tengo"
 			}
 
+			if !filepath.IsAbs(c.importDir) {
+				if p, err := filepath.Abs(c.importDir); err == nil {
+					c.importDir = p
+				}
+			}
```

Comment:
I think `c.importDir` should be resolved to absolute path when it's updated (for example, inside SetImportDir function).

Reply:
Yes, it is better.

Reply:
If user does not call `Script.SetImportDir`, we have an empty `Compiler.importDir`, I should check if it is absolute in compiler in any way, right? I will just change `Script.SetImportDir(string)` to `Script.SetImportDir(string) error`, right?

Reply:
I think so. RE empty import path, I haven't tested it but I think `filepath.Join("", moduleName)` might work using the current working directory... Please test it.